src/react/PropTypes.js

```diff
@@ -1,14 +1,12 @@
-import {PropTypes} from "react";
+import PropTypes from "prop-types";
 
```

Comment:
`PropTypes` in later version of `react` come from the `prop-types` package.  https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html